docs/community/insights.md

```diff
@@ -5,9 +5,17 @@ sidebar_label: 'Insights'
 keywords:
 - insights
 ---
-
- The Insights feature is where you pick and view the repositories that you often contribute to. It's a great way to stay updated on their growth.
+The Insights feature is your one-stop solution for understanding the pulse of your open-source projects and contributions. This feature is designed to provide a comprehensive view of your project's health and your individual contributions. From contribution trends to community health, OpenSauced Insights helps you make data-driven decisions that align with your goals. It's not just about numbers; it's about actionable intelligence that can redefine your approach to open source.
 
 ![insight-pages-demo](../../static/gif/insight-page-demo.gif)
 
-Craving for more? Check out the [Discovering Open-Source with OpenSauced](https://github.com/open-sauced/intro/blob/main/05-how-to-contribute-to-open-source.md#discovering-open-source-with-opensauced) section in our free beginner course on open source.
+## How to use Insights
+### Contributors
+- Track your contributions and the impact you're making;
+- Find new projects to contribute to;
+- Discover other contributors to collaborate with.
+
+### Maintainers
+- Track the health of your project;
+- Identify contributors who are making an impact;
```

Comment:
[Proofreading] Since the second bullet point in the Contributors list ends with a period, it’s best to use this approach for the rest of this section to stay consistent. 
```suggestion
- Track your contributions and the impact you're making.
- Find new projects to contribute to.
- Discover other contributors to collaborate with.

### Maintainers
- Track the health of your project.
- Identify contributors who are making an impact.
```